### ISSUE TITLE
Crashes regularly with response error

### ISSUE THREAD

**Issue by dpriskorn** [egils-consulting/asseeibot#1]
this is because of upstream error
https://github.com/aio-libs/aiohttp/issues/3904
It should be ignored, if anyone know how to do that, please comment.

**Comment by dpriskorn** [comment]
This could be a possible solution https://github.com/aio-libs/aiohttp/issues/3904#issuecomment-759205696

**Comment by dpriskorn** [comment]
```
WARNING:asyncio:Executing <Task pending name='Task-1' coro=<main() running at /home/egil/src/python/asseeibot/asseeibot/main.py:180> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fca53f8e7c0>()] created at /usr/lib/python3.9/asyncio/base_events.py:424> cb=[_run_until_complete_cb() at /usr/lib/python3.9/asyncio/base_events.py:184] created at /usr/lib/python3.9/asyncio/base_events.py:621> took 3.813 seconds
Traceback (most recent call last):
  File "/home/egil/src/python/asseeibot/asseeibot/main.py", line 226, in <module>
    loop.run_until_complete(main())
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/egil/src/python/asseeibot/asseeibot/main.py", line 180, in main
    async for event in aiosseclient(
  File "/usr/lib/python3.9/site-packages/aiosseclient.py", line 24, in aiosseclient
    async for line in response.content:
  File "/usr/lib/python3.9/site-packages/aiohttp/streams.py", line 39, in __anext__
    rv = await self.read_func()
  File "/usr/lib/python3.9/site-packages/aiohttp/streams.py", line 338, in readline
    await self._wait("readline")
  File "/usr/lib/python3.9/site-packages/aiohttp/streams.py", line 306, in _wait
    await waiter
aiohttp.client_exceptions.ClientPayloadError: Response payload is not completed
ERROR:asyncio:Unclosed response
client_response: <ClientResponse(https://stream.wikimedia.org/v2/stream/recentchange) [200 OK]>
<CIMultiDictProxy('Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Headers': 'accept, x-requested-with, content-type', 'Access-Control-Expose-Headers': 'etag', 'x-xss-protection': '1; mode=block', 'x-content-type-options': 'nosniff', 'x-frame-options': 'SAMEORIGIN', 'content-security-policy': "default-src 'none'; script-src 'self' 'unsafe-inline'; connect-src *; style-src 'self' 'unsafe-inline'; img-src 'self'; font-src 'self'", 'x-content-security-policy': "default-src 'none'; script-src 'self' 'unsafe-inline'; connect-src *; style-src 'self' 'unsafe-inline'; img-src 'self'; font-src 'self'", 'x-webkit-csp': "default-src 'none'; script-src 'self' 'unsafe-inline'; connect-src *; style-src 'self' 'unsafe-inline'; img-src 'self'; font-src 'self'", 'Content-Type': 'text/event-stream; charset=utf-8', 'Cache-Control': 'no-cache', 'Date': 'Thu, 08 Apr 2021 12:20:26 GMT', 'Server': 'eventstreams-tls', 'Age': '0', 'X-Cache-Int': 'cp4027 miss', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive')>

source_traceback: Object created at (most recent call last):
  File "/home/egil/src/python/asseeibot/asseeibot/main.py", line 226, in <module>
    loop.run_until_complete(main())
  File "/usr/lib/python3.9/asyncio/base_events.py", line 629, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.9/asyncio/base_events.py", line 596, in run_forever
    self._run_once()
  File "/usr/lib/python3.9/asyncio/base_events.py", line 1882, in _run_once
    handle._run()
  File "/usr/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/egil/src/python/asseeibot/asseeibot/main.py", line 180, in main
    async for event in aiosseclient(
  File "/usr/lib/python3.9/site-packages/aiosseclient.py", line 22, in aiosseclient
    response = await session.get(url, **kwargs)
  File "/usr/lib/python3.9/site-packages/aiohttp/client.py", line 542, in _request
    resp = await req.send(conn)
  File "/usr/lib/python3.9/site-packages/aiohttp/client_reqrep.py", line 674, in send
    self.response = response_class(
```

**Comment by dpriskorn** [comment]
I fixed this in v0.1-alpha0 by catching the exception and encasing the whole thing in a while loop.
